.gitignore

```diff
@@ -36,3 +36,5 @@ src/peilbeheerst_model/.vscode
 notebooks/rijkswaterstaat/plots/
 /src/peilbeheerst_model/peilbeheerst_model/Oud
 /src/peilbeheerst_model/ribasim
+
+.env
```

Comment:
```suggestion
.env.default
```

Should it be this?

Reply:
No, the .env is secret (cloud password), so should never be checked in. We provide an .example so we know which envs we can/should set. I renamed it to .default because of some weird Quarto behavior.